.github/workflows/test.yml

```diff
@@ -28,6 +39,31 @@ jobs:
             echo "Run 'go mod tidy' and push it"
             exit 1
           fi
+      - name: Install plugin from repository
+        run: |
+          trivy plugin install github.com/dramf/md
```

Comment:
can we use commit number here?

Reply:
updated in https://github.com/dramf/md/pull/6/commits/abdc98d88c7961f02697bb5610aa53fbe4dc70a1